docs/data/base/components/form-control/UseFormControl.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import FormControlUnstyled, {
+  useFormControlUnstyled,
+} from '@mui/base/FormControlUnstyled';
+
+function CustomInput() {
```

Comment:
While this demo nicely shows the usage, I think it would be useful to create a more "real life" example maybe using the `InputUnstyled` too.

Reply:
When I say more "real life" I mean input with some validation and error message.